src/Reduce.php

```diff
@@ -154,4 +154,36 @@ public static function toString(iterable $data, string $separator = '', string $
         $joined = \implode($separator, $items);
         return $prefix . $joined . $suffix;
     }
+
+    /**
+     * Reduces given collection to array of its upper and lower bounds.
+     *
+     * Returns [null, null] if given collection is empty.
+     *
+     * @param iterable<numeric> $numbers
+     *
+     * @return array{numeric, numeric}|array{null, null}
+     */
+    public static function toBounds(iterable $numbers): array
```

Comment:
Suggestion: Since `toMin` and `toMax` are already present, for consistency what do you think about naming this `toMinMax`. It also implies there are two values. "To bounds" on the other hand could be confused with restricting or adjusting something to be within some predefined bounds condition perhaps.

Reply:
Done.